frontend/src/utils/inRange.ts

```diff
@@ -0,0 +1 @@
+export const inRange = (x: number, min: number, max: number) => (x - min) * (x - max) <= 0;
```

Comment:
Consider put this function in the relevant file / directory.
I don't think other components will use it, it can be a private function under `hybridPaginationHook.ts`

Reply:
- moved fn back to file ([8c98191](https://github.com/origranot/reduced.to/pull/396/commits/8c981913d2c26e33af922bf66966254f7af3f825))